lib/validator/formchecker.cpp

```diff
@@ -38,7 +38,7 @@ void FormChecker::reset(bool CleanGlobal) {
     Tables.clear();
     Mems = 0;
     Globals.clear();
-    Datas.clear();
+    Data.clear();
```

Comment:
I would say this breaks the name convention here, maybe need more consideration

Reply:
`data` is the plural form of `datum`. So we don't need to use `datas` to represent multiple data.

Reply:
`Data` here means `DataSegment.` So it's fine to use `Datas`.
We remove all of the `Segment` to simplify the naming.